.github/actions/docker-build/action.yaml

```diff
@@ -0,0 +1,27 @@
+name: docker-build
+description: Reusable action to build repos' docker images
+
+runs:
+  using: composite
+  steps:
+    - name: ci/prepare-docker-environment
+      uses: ./.github/actions/docker-prepare
+
+    # - name: cd/scan-docker-security
+    #   uses: aquasecurity/trivy-action@062f2592684a31eb3aa050cc61e7ca1451cecd3d # v0.18.0
+    #   with:
+    #     image-ref: "mattermost/atlantis"
+    #     format: "table"
+    #     ignore-unfixed: true
+    #     vuln-type: "os,library"
+    #     severity: "CRITICAL"
```

Comment:
@andrleite is this commented code needed?

Reply:
I'll uncomment it when we have the first image in the registry. Right now we don't have any image mattermost/atlantis so it is failing.